genai-perf/genai_perf/inputs/converters/tensorrtllm_engine_converter.py

```diff
@@ -82,4 +82,35 @@ def _add_request_params(self, payload: Dict, config: InputsConfig) -> None:
                 payload["min_length"] = [num_tokens]
 
         for key, value in config.extra_inputs.items():
-            payload[key] = [value]
+            if key == "set_end_id":
+                payload["end_id"] = [config.tokenizer._tokenizer.eos_token_id]
+            elif key == "apply_chat_template":
+                pass
+            else:
+                payload[key] = [value]
+
+    def _encode_tokens(self, prompt: str, config: InputsConfig) -> List[int]:
+        if config.extra_inputs.__contains__("apply_chat_template"):
```

Comment:
```suggestion
        if "apply_chat_template" in config.extra_inputs:
```
Is this a viable alternative?

Reply:
I was about to suggest the same :)

Reply:
Great suggestion, fixed!